src/i18n/nl_NL/buildings.ts

```diff
@@ -565,6 +597,23 @@ export default {
                 givesParkingLots: 0,
                 unlocksVehicleTypes: [16, 30, 52, 63],
             },
+            ...multiplyExtension(
+                {
+                    caption: 'Haakarmbak parkeerplaats',
+                    credits: 100_000,
+                    coins: 20,
+                    duration: '7 Dagen',
+                    isVehicleExtension: true,
+                    givesParkingLots: 1,
+                    unlocksVehicleTypes: [26, 27, 29, 32, 45, 51, 61],
+                    parkingLotReservations: [[27, 29, 32, 45, 51, 61]],
+                    cannotDisable: true,
+                },
+                2
+            ),
```

Comment:
1. function overload with 3x "null"
2. "Null" as multiplier == extension disabled?

Reply:
1. not sure if the helper function `multiplyExtension` will work with `null`, that's why added as 3 values
2. `null` means extension not available or ID not used ingame. Example: 
![grafik](https://user-images.githubusercontent.com/27238105/171993682-6983ccef-c211-4cc1-afe4-106b01af6086.png)